src/Components/missions/Missions.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { toggleJoining } from '../../redux/Missions/missionsReducer';
+import style from './missions.module.css';
+
+function Missions() {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+  return (
+    <>
+      <table className={style.table} data-testid="list-of-missions">
+        <thead className={style.thead}>
+          <tr className={style.header}>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th> </th>
+          </tr>
+        </thead>
+        {missions.map((el) => (
+          <tbody key={el.id} className={style.item}>
```

Comment:
- [ ] I think it's better to create striped lines for your table. Will create a better UI and makes your page look better. For more information [click here](https://www.w3schools.com/howto/howto_css_table_zebra.asp).